packages/cli/src/lib/transform.ts

```
@@ -3,10 +3,14 @@ import debug from "debug";
 import fs from "fs";
 import path from "path";
 import { TransformOptions } from "./transform-options";
+import { fileURLToPath } from "url";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)